test/test_formats.py

```diff
@@ -146,13 +147,14 @@ class FormatsTest(unittest.TestCase):
 
         def setUp(self):
             self.tags = TAGS.copy()
-            self.setup_tags()
+            _, self.testfile_ext = os.path.splitext(self.testfile)
```

Comment:
Just a minor thing, using `_` as place holder for unused value is pretty standard in Python, but since we are using `_()` for gettext/i18n, it is a bad idea to overwrite it.
Please use something else (like `root`, used in `os.path.splitext()` documentation, but you lose the unused info, or just `unused`)